glusterd2/commands/volumes/volume-options-get.go

```diff
@@ -78,7 +78,7 @@ func createVolumeOptionsGetResp(volinfo *volume.Volinfo) *api.VolumeOptionsGetRe
 		// now return all options
 		for _, opt := range xl.Options {
 			for _, k := range opt.Key {
-				optName := xl.ID + "." + k
+				optName := xl.Category + "/" + xl.ID + "." + k
```

Comment:
The change is good for most cases. But if a option is set with the prefix of template name then that will not be catch. For example `glustercli volume set gv1 brick.changelog.rollover-time 30`. This will be stored in volinfo as `brick.features/changelog.rollover-time`. In this case above comparison is not sufficient. `createVolumeOptionGetResp` function needs modification to handle this case.

Reply:
@rishubhjain can you please refresh the PR with the suggested changes?